install-dev/data/xml/tab.xml

```diff
@@ -439,6 +439,18 @@
                 <tab id="Multistore" id_parent="Advanced_Parameters" active="0" enabled="1" hide_host_mode="0" icon="">
                     <class_name>AdminShopUrl</class_name>
                 </tab>
+                <tab id="Security" id_parent="Advanced_Parameters" active="1" enabled="1" hide_host_mode="0" icon="">
+                    <class_name>AdminParentSecurity</class_name>
+                </tab>
+                    <tab id="Security_1" id_parent="Security" active="1" enabled="1" hide_host_mode="0" icon="">
```

Comment:
indentation bit off

Reply:
It's normal, if you look the whole file, I did the same thing as they did (don't ask me why it's like that :sweat_smile: )